src/lib/gql/fetcher.tsx

```diff
@@ -103,6 +103,21 @@ export const getConfigPage = cache(
   }
 )
 
+export const getConfigPageField = async <T extends ConfigPagesUnion, F>(
```

Comment:
@pookmish is this okay for us to copypasta over and use? Any concerns with making this compatible or should it be okay to use out of box? 🤔

Reply:
To my knowledge, it should be safe to pull it over from the other repos.